css-two-column-layout/styles.css

```diff
@@ -0,0 +1,157 @@
+.container {
+  max-width: 1346px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.heading-row {
```

Comment:
since all of the row classes have the same CSS, I would recommend making a single `row` class and putting that on each row element